### PR TITLE
:construction_worker: Website Footer #135

### PR DESCRIPTION
**Pull request by kiplangat01**
# Description

Created a  a simple footer that displays my information in a structured way.

Reference the issue related to this PR as shown below. Every issue has it's own number you can find the issue numbers [#135](https://github.com/italanta/italanta-apps/issues).

Fixes issue #135 

## Screenshot 
Screenshot from 2023-08-31 22-24-00
(1)Screenshot from 2023-08-31 22-43-25
(2)Screenshot from 2023-08-31 22-43-30

## Checklist

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by Beulah-Matt — COMMENTED** [review]
Good attempt but I went with PR #144 for this issue

**Comment by kiplangat01** [comment]
> Good attempt but I went with PR #144 for this issue

no problem

File: libs/elements/layout/footer/src/lib/components/footer/footer.component.html [hunk 1]
```diff
@@ -0,0 +1,55 @@
+<section>
+    <ng-content select="[button], [head] "></ng-content>
```
Comment (LemmyMwaura) [inline libs/elements/layout/footer/src/lib/components/footer/footer.component.html:2]: The data to be dynamically rendered is passed as only one item

**Review by LemmyMwaura — COMMENTED** [review]
This references more than one issue(code). Kindly work on one issue at a time.

**Comment by LemmyMwaura** [comment]
Closing this as it's already been resolved

**Pull request closed without merging**